includes/emails/functions.php

```diff
@@ -43,6 +43,7 @@ function edd_email_purchase_receipt( $payment_id, $admin_notice = true, $to_emai
 
 	$heading      = edd_get_option( 'purchase_heading', __( 'Purchase Receipt', 'easy-digital-downloads' ) );
 	$heading      = apply_filters( 'edd_purchase_heading', $heading, $payment_id, $payment_data );
+	$heading      = edd_do_email_tags( $heading, $payment_id );
```

Comment:
This needs to be done on _all_ emails, not just receipts, so it should be done via the EDD_Emails class.